features/open_syspaste.py

```diff
@@ -1,14 +1,19 @@
+'''
+Load new table from system clipboard
+'''
+
 from visidata import vd, BaseSheet, Path
 
 
 @BaseSheet.api
-def open_syspaste(sheet, filetype='tsv'):
+def open_syspaste(sheet, filetype=None):
     import io
 
+    filetype = filetype or vd.input("paste as filetype: ", value='tsv')
```

Comment:
Can we move this `input()` to the execstr?  The menu system looks for the string `input` and that's how it figures out that the command will require input.  (It's cosmetic only, but still how all commands are done).

Reply:
Should this be changed in `clipboard.py` too? (I copied this from there.)

Reply:
Ah, probably!  Sorry for the confusion.  Feel free to add to this PR in a separate commit, or I can fix it in a bit.

Reply:
I've pushed a new commit with this change for `open-syspaste`. I might leave the changes to `clipboard.py` for you, as `vd.input` in `syscopyValue()` refers to the sheet row type, and I'm not sure if that's known when the execstr is defined.

Reply:
Ah, okay.  I looked at it, and I'll leave it like it is for the time-being.  If we do an overhaul of the clipboard as mentioned in another thread, we can try to clean that up then.  Maybe we can figure out how to design away the input()s entirely.